pkg/orchestration/wiring/wiringutil/knownshapes/labelled.go

```diff
@@ -0,0 +1,39 @@
+package knownshapes
+
+import (
+	smith_v1 "github.com/atlassian/smith/pkg/apis/smith/v1"
+	"github.com/atlassian/voyager/pkg/orchestration/wiring/wiringplugin"
+)
+
+const (
+	LabelledShape wiringplugin.ShapeName = "voyager.atl-paas.net/LabelledShape"
```

Comment:
I'm not sure what the meaning here is. What would you write as a doc here?

Reply:
I didn't know how to best approach this. I think this is interesting case. Let's discuss f2f please.